examples/deploy/terraform/cluster/main.tf

```diff
@@ -28,7 +28,10 @@ module "eks" {
   bastion_info        = local.infra.bastion
   create_eks_role_arn = local.infra.create_eks_role_arn
   tags                = local.infra.tags
-
+  privatelink = {
+    enabled           = false
```

Comment:
should this be 
`enabled = var.enable_private_link`

Reply:
shoudnt `route53_hosted_zone_name` be part of the `privatelink` so all `privatelink` vars be validated

Reply:
I didnt get this, if I start adding the env vars I am replicating the same but in examples, is that the intention?

Reply:
My mistake, what I meant here you covered elsewhere.  Given that this example is meant to be consume by external users and private link is off by default. I think we should omit this entry here.

Reply:
fixed